android/app/build.gradle

```diff
@@ -5,7 +5,6 @@ plugins {
 }
 
 group = 'com.github.bitcoin-portal'
-version = '0.13.2'
```

Comment:
@percivalruiz-bitcoin @minh-rakuten what is the reason for removing versioning? 

We should import the library by version tags instead of trying to figure out what the commits are/do

Reply:
I think we use the version tags from here https://github.com/bitcoin-portal/asset-converter/releases and no need to declare the version in gradle.

Reply:
This one [0.13.2.b1 ](https://github.com/bitcoin-portal/asset-converter/releases/tag/0.13.2.b1)without the version  is fetchable from github.

The `version` does not associate this repo code to tagged version either -> you still have to check git commit to see what changes between old/new versions.

To make that `version` really connected to the release then probably we must have a CI step to check the `version` in gradle and publish a new tag automatically, not doing manually like this https://github.com/bitcoin-portal/asset-converter/pull/23 for certainty.

Reply:
I approved it, we can try it by publishing with git commits as release notes for the time being.